### ISSUE TITLE
Could not connect to API with id: default: 3099 - ERROR3099 - Some Wazuh daemons are not ready in node 'node01' (wazuh-modulesd->failed)

### ISSUE THREAD

**Issue by NitroCao** [wazuh/wazuh#8359]
|Wazuh version|Component|
|---|---|
| 3.13.2-1 | wazuh-manager |
| 3.13.2-1 | wazuh-api |

OS: CentOS 7
When opening Kibana Wazuh app, it shows the error `Could not connect to API with id: default: 3099 - ERROR3099 - Some Wazuh daemons are not ready in node 'node01' (wazuh-modulesd->failed)`.  
Check the log on the server, I got the message:
```
[13477051.189472] wazuh-modulesd[25152]: segfault at 0 ip 0000000000462cf9 sp 00007f55956dfa40 error 4 in wazuh-modulesd[400000+dd000]
[13515826.715491] wazuh-modulesd[2627]: segfault at 0 ip 0000000000462cf9 sp 00007f483fe16a40 error 4 in wazuh-modulesd[400000+dd000]
```
`wazuh-modulesd` doesn't run. When I restart wazuh-manager, wazuh-modulesd will terminate after a while.

**Comment by Selutario** [comment]
Hello @NitroCao!

Unfortunately, there is a critical bug that causes the Wazuh manager processes to crash (Wazuh agents are not affected). More specifically, it affects Wazuh managers 3.13 and later versions.

This bug is caused by a problem in the Vulnerability Detection module. Although the bug has already been fixed with the release of [Wazuh 4.1.5](https://documentation.wazuh.com/current/release-notes/release_4_1_5.html), the team is still working on a release for Wazuh 3.X that will fix the problem for said version. Temporarily, the problem can be solved by disabling the NVD provider on the manager configuration file. Vulnerability Detector won’t work, but the rest of the manager capabilities will work normally.

To apply the temporary fix, on the Wazuh manager system, you need to edit your `/var/ossec/etc/ossec.conf` file and disable the NVD provider. This needs to be done inside the <vulnerability-detection> section of the file:
```XML
<provider name="nvd">
  <enabled>no</enabled>
  <update_from_year>2010</update_from_year>
  <update_interval>1h</update_interval>
</provider>
```

Then, to apply changes, you will need to restart your Wazuh manager:
```
systemctl restart wazuh-manager
```

Please make sure the manager is properly working afterwards, by checking that the wazuh-modulesd process is up and running in your system:
```
ps aux | grep -i wazuh-modulesd
```

We will be releasing a patched version for 3.X early this week. With this patch the Vulnerability Detector module will work well again.

Apologies for the inconvenience. Let us know if you have any questions.

Regards,
Selu.

**Comment by Selutario** [comment]
Hi again @NitroCao,

Wazuh v3.13.3 has been released with a fix for the Vulnerability Detector bug. Check it out here:
https://documentation.wazuh.com/3.13/release-notes/release_3_13_3.html

I'm closing the issue. Do not hesitate to reopen it or to open another if you have more questions. 

Regards,
Selu.
